eth/jobmanager.go

```diff
@@ -40,11 +40,16 @@ func (j *JobManager) Start() error {
 		for {
 			select {
 			case l := <-logsCh:
-				jid, strmID, _, tAddr, err := GetInfoFromJobEvent(l, j.c)
+				jid := new(big.Int).SetBytes(l.Data[0:32])
```

Comment:
Do we have some kind of serialization/deserialization logic to handle parsing the logs?